KeenSwiftClientExample/KeenSwiftClientExample/AppDelegate.swift

```diff
@@ -8,19 +8,30 @@
 
 import UIKit
 
+class ExampleLogger: KeenLogSink {
```

Comment:
Should be pretty simple, but wouldn't be a big deal to do. If we do so I'd suggest some project refactoring of the existing projects so that they have their own workspaces and use cocoapods. We could either do a new project alongside those, or add on to them.